pkg/sdk/tasks_gen_test.go

```diff
@@ -552,3 +554,66 @@ func TestTasks_Execute(t *testing.T) {
 		assertOptsValidAndSQLEquals(t, opts, "EXECUTE TASK %s RETRY LAST", id.FullyQualifiedName())
 	})
 }
+
+func TestParseTaskSchedule(t *testing.T) {
+	testCases := map[string]struct {
+		Schedule             string
+		ExpectedTaskSchedule *TaskSchedule
+		Error                string
+	}{
+		"valid schedule: m minutes": {
+			Schedule:             "5 m",
+			ExpectedTaskSchedule: &TaskSchedule{Minutes: 5},
+			Error:                "",
+		},
+		"valid schedule: M minutes": {
+			Schedule:             "5 m",
+			ExpectedTaskSchedule: &TaskSchedule{Minutes: 5},
+			Error:                "",
```

Comment:
`Error` should be omitted in such cases.

Reply:
True